patchwork/templates/patchwork/partials/series-list.html

```diff
@@ -0,0 +1,154 @@
+{% load person %}
```

Comment:
This doesn't need to be a partial. The only reason we have a `patch-list` partial is because we re-use that partial for both the patch list view and the bundle view (bundles are just lists of patches). You can combine this into `series.html`.